test/functional/page_objects/visual_builder_page.ts

```diff
@@ -35,21 +35,6 @@ export class VisualBuilderPageObject extends FtrService {
   private readonly timePicker = this.ctx.getPageObject('timePicker');
   private readonly visChart = this.ctx.getPageObject('visChart');
 
-  public async resetPage(
```

Comment:
Won't make more sense to rewrite it to something like and reuse it?

```ts
public async resetPage({timeOnly: boolean}){
  if(!timeOnly){
    await this.navigateToNewVisualization();
      await this.clickVisualBuilder();
      await this.checkVisualBuilderIsPresent();
  }
      await visualBuilder.setTime();
}

Reply:
I would like to keep the setTime separately if this is the only function that needs to run but I did some refactoring to the resetPage and I am using it where possible https://github.com/elastic/kibana/pull/156785/commits/74e59b249c09a3312105440677faf26c89a671c8